qa/ccs-common-rest/src/yamlRestTest/resources/roles.yml

```diff
@@ -0,0 +1,11 @@
+remote_search_role:
```

Comment:
It should work because the test cluster explicitly requires the feature flag to be turned on. 

That said, this made me find possibly a bug for the new test cluster infrastructure. It does not seem to work with release build correctly. I'll bring it to the delivery team. Thanks!